automation/utils/list_manip.py

```diff
@@ -1,8 +1,24 @@
+from collections import Iterable
```

Comment:
Leaving comment on Iterable as it broke an instance of pulling Encounters, not sure if it's a module we need to implement/add or if we can ignore and replace this code.